src/lib/core/hooks/computeNumberDateDefaultDependentAxisRange.ts

```diff
@@ -22,6 +24,17 @@ export function useDefaultDependentAxisRange(
 ): NumberOrDateRange | undefined {
   // find max of stacked array, especially with overlayVariable
   const defaultDependentAxisRange = useMemo(() => {
+    // explicitly check empty data
+    if (
+      (!isFaceted(data?.value?.dataSetProcess) &&
+        data?.value?.dataSetProcess.series == null) ||
+      (isFaceted(data?.value?.dataSetProcess) &&
+        data?.value?.dataSetProcess?.facets != null &&
```

Comment:
For my education, what is the value of this check that the facets != null? Is it simply to ensure we're getting a good answer about the length === 0 check in the next line?